### DISCUSSION TITLE
Ignore error raised by renovate-config-validator

### DISCUSSION THREAD

**Discussion by mueller-ma** [renovatebot/renovate#23976] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

On my self-hosted GitLab I want Renovate to access GitLab's container registry. I added the following code to config.js:

````js
  "hostRules": [
    {
      "matchHost": "https://gitlab.example.com",
      "hostType": "docker",
      "username": "renovate",
      "password": process.env.RENOVATE_TOKEN
    }
  ]
````

The password is stored as private CI variable, so it doesn't need to written in plain text anywhere. It's working fine, but the validator complains about it:

````
$ renovate-config-validator --strict
INFO: Validating config.js
ERROR: Found errors in configuration
       "file": "config.js",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "Configuration option `hostRules[0].password` should be a string"
         }
       ]
````

How can I tell renovate-config-validator to ignore this issue? Or can an exception be added to the validator as most users would like to keep their tokens secret?

### What you tried so far.

Search in the docs and GitHub issue on how to ignore issues by renovate-config-validator.

**Comment by rarkins** [comment, marked as answer]
Does `RENOVATE_TOKEN=dummy renovate-config-validator --strict` work?

**Reply by mueller-ma** [reply]
Thanks!
